test/src/api/responderTest.php

```diff
@@ -11972,6 +11972,95 @@ public function test_interface_game_040() {
+        $expData['gameActionLogCount'] = 2;
+        $expData['gameState'] = 'START_TURN';
+        $expData['activePlayerIdx'] = 0;
+        $expData['playerWithInitiativeIdx'] = 0;
+        $expData['validAttackTypeArray'] = array('Power', 'Skill');
+        $expData['playerDataArray'][0]['activeDieArray'][0]['value'] = 1;
+        $expData['playerDataArray'][0]['activeDieArray'][1]['value'] = 1;
+        $expData['playerDataArray'][0]['activeDieArray'][2]['value'] = 3;
+        $expData['playerDataArray'][0]['activeDieArray'][3]['value'] = 9;
+        $expData['playerDataArray'][0]['activeDieArray'][4]['value'] = 4;
+        $expData['playerDataArray'][0]['activeDieArray'][4]['sides'] = 18;
+        $expData['playerDataArray'][0]['activeDieArray'][4]['description'] = 'Stealth Y Swing Die (with 18 sides)';
+        $expData['playerDataArray'][1]['activeDieArray'][0]['value'] = 4;
+        $expData['playerDataArray'][1]['activeDieArray'][1]['value'] = 19;
+        $expData['playerDataArray'][1]['activeDieArray'][2]['value'] = 8;
+        $expData['playerDataArray'][1]['activeDieArray'][3]['value'] = 11;
+        $expData['playerDataArray'][1]['activeDieArray'][4]['value'] = 2;
+        $expData['playerDataArray'][1]['activeDieArray'][5]['value'] = 14;
+        $expData['playerDataArray'][0]['roundScore'] = 22;
+        $expData['playerDataArray'][1]['roundScore'] = 60;
+        $expData['playerDataArray'][0]['sideScore'] = -25.3;
+        $expData['playerDataArray'][1]['sideScore'] = 25.3;
+
+        $retval = $this->verify_api_loadGameData($expData, $gameId, 10);
+
+        /////
+        // Move 2: responder003 chose to perform a Skill attack using [k(8):3,d(Y=18):4] against [(20):4]; responder003 must turn down fire dice to complete this attack.
+        $this->verify_api_submitTurn(
+            array(),
+            'responder003 chose to perform a Skill attack using [k(8):3,d(Y=18):4] against [(20):4]; responder003 must turn down fire dice to complete this attack. ',
+            $retval, array(array(0, 2), array(0, 4), array(1, 0)),
+            $gameId, 1, 'Skill', 0, 1, '');
+
+        array_unshift($expData['gameActionLog'], array('timestamp' => 'TIMESTAMP', 'player' => 'responder003', 'message' => 'responder003 chose to perform a Skill attack using [k(8):3,d(Y=18):4] against [(20):4]; responder003 must turn down fire dice to complete this attack'));
+        $expData['gameActionLogCount'] += 1;
+        $expData['gameState'] = 'ADJUST_FIRE_DICE';
+        $expData['validAttackTypeArray'] = array('Skill');
+        $expData['playerDataArray'][0]['activeDieArray'][2]['properties'] = array('IsAttacker');
+        $expData['playerDataArray'][0]['activeDieArray'][4]['properties'] = array('IsAttacker');
```

Comment:
You accidentally removed the:
```php
$retval = $this->verify_api_loadGameData($expData, $gameId, 10);
```

from this test, so the end of it is not actually doing anything.